toolkit/tools/imagecustomizer/docs/verity.md

```diff
@@ -0,0 +1,141 @@
+# Verity Image Recommendations
+
+The Verity-enabled root filesystem is always mounted as read-only. Its root hash
+and hash tree are computed at build time and verified by systemd during the
+initramfs phase on each boot. When enabling the Verity feature, it is
+recommended to create a writable persistent partition for any directories that
+require write access. Critical files and directories can be redirected to the
+writable partition using symlinks or similar methods.
+
+Please also note that some services and programs on Azure Linux may require
+specific handling when using Verity. Depending on user needs, there are
+different configuration options that offer tradeoffs between convenience and
+security. Some configurations can be made flexible to allow changes, while
+others may be set as immutable for enhanced security.
+
+## systemd-growfs-root
+
+This service attempts to resize the root filesystem, which fails since verity
+makes the root filesystem readonly and a fixed size.
+
+### Solution 1: Do nothing
+
+Since the root filesystem is readonly, the `systemd-growfs-root` service will
+fail. However, the only impact will be an error in the boot logs.
+
+### Solution 2: Disable service
+
+Disabling the service removes the error from the boot logs.
+
+```yaml
+os:
+  services:
+    disable:
+    - systemd-growfs-root
+```
+
+## cloud-init-local, cloud-config, cloud-final
```

Comment:
nit: Just `## cloud-init`

While there are multiple systemd services included with cloud-init, they are all part of the same application.

Reply:
Fixed. Thanks.